src/mumble/Database.cpp

```diff
@@ -238,6 +241,26 @@ void Database::setLocalIgnored(const QString &hash, bool ignored) {
 	execQueryAndLogFailure(query);
 }
 
+bool Database::isLocalIgnoredTTS(const QString &hash) {
+	QSqlQuery query(db);
+
+	query.prepare(QLatin1String("SELECT `hash` FROM `ignored_tts` WHERE `hash` = ?"));
+	query.addBindValue(hash);
+	execQueryAndLogFailure(query);
+	return query.next();
+}
+
+void Database::setLocalIgnoredTTS(const QString &hash, bool ignoredTTS) {
+	QSqlQuery query(db);
+
+	if (ignoredTTS)
+		query.prepare(QLatin1String("INSERT INTO `ignored_tts` (`hash`) VALUES (?)"));
```

Comment:
Doesn't this allow for multiple entries in the table for the same `hash`?

Reply:
No, there is a UNIQUE constraint on the index.